docs/src/writing-tests-js.md

```diff
@@ -113,22 +101,22 @@ learn more about them.
 | [`method: Locator.selectOption`] | Select option in the drop down |
 
 ## Assertions
-  
+
 Playwright includes [test assertions](./test-assertions.md) in the form of `expect` function. To make an assertion, call `expect(value)` and choose a matcher that reflects the expectation.
 
-There are many generic matchers like `toEqual`, `toContain`, `toBeTruthy` that can be used to assert any conditions.
+Playwright includes generic matchers like `toEqual`, `toContain`, `toBeTruthy` that can be used to assert any conditions.
```

Comment:
I think it's better to move the section about generic matchers to be after the list of web-first assertions.